server/main.lua

```diff
@@ -13,136 +14,107 @@ function ExploitBan(id, reason)
+	exports['Renewed-Banking']:addAccountMoney(account, amount, 'qbx_management')
 end
 
 ---@param account string
 ---@param amount number
 ---@return boolean
 function RemoveMoney(account, amount)
-
 	if amount <= 0 then return false end
-    if not Accounts[account] then
-        Accounts[account] = 0
-    end
-
-    local isRemoved = false
-
-    if Accounts[account] >= amount then
-        Accounts[account] -= amount
-        isRemoved = true
-    end
-
-	exports['Renewed-Banking']:removeAccountMoney(account, amount)
-	return isRemoved
+	return exports['Renewed-Banking']:removeAccountMoney(account, amount, 'qbx_management')
 end
 
-MySQL.ready(function ()
-	local funds = MySQL.query.await('SELECT job_name, amount FROM management_funds')
-	if not funds then return end
-
-	for i = 1, #funds do
-		local v = funds[i]
-		Accounts[v.job_name] = v.amount
-	end
-end)
-
 ---@param src number
 ---@param amount number
----@param pDataType 'job'|'gang'
----@param type 'boss'|'gang'
+---@param type 'job'|'gang'
 ---@param reason string
 ---@return boolean success
 ---@return Player? player populated if successful
 ---@return string? accountName populated if successful
-function WithdrawMoney(src, amount, pDataType, type, reason)
+function WithdrawMoney(src, amount, type, reason)
 	local player = exports.qbx_core:GetPlayer(src)
 
-	if not player.PlayerData[pDataType].isboss then ExploitBan(src, 'withdrawMoney Exploiting') return false end
+	if not player.PlayerData[type].isboss then ExploitBan(src, 'withdrawMoney Exploiting') return false end
 
-	local account = player.PlayerData[pDataType].name
-	if RemoveMoney(account, amount, type) then
-        player.Functions.AddMoney("cash", amount, reason)
-		exports.qbx_core:Notify(src, "You have withdrawn: $" ..amount, "success")
+	local account = player.PlayerData[type].name
+	if RemoveMoney(account, amount) then
+        player.Functions.AddMoney('cash', amount, reason)
+		local logArea = type == 'gang' and 'gang' or 'boss'
+		exports.qbx_core:Notify(src, 'You have withdrawn: $' ..amount, 'success')
+		TriggerEvent('qb-log:server:CreateLog', logArea..'menu', 'Withdraw Money', 'orange', player.PlayerData.charinfo.firstname .. ' ' .. player.PlayerData.charinfo.lastname .. ' successfully withdrew $' .. amount .. ' (' .. account .. ')', false)
         return true, player, account
 	else
-		exports.qbx_core:Notify(src, "You dont have enough money in the account!", "error")
+		exports.qbx_core:Notify(src, 'You dont have enough money in the account!', 'error')
         return false, player, account
     end
 end
 
 ---@param src number
 ---@param amount number
----@param pDataType 'job'|'gang'
----@param type 'boss'|'gang'
+---@param type 'job'|'gang'
 ---@return boolean success
 ---@return Player? player populated if successful
 ---@return string? accountName populated if successful
-function DepositMoney(src, amount, pDataType, type)
+function DepositMoney(src, amount, type)
```

Comment:
```suggestion
local function DepositMoney(src, amount, type)
```

Since we are going to 1 resource file we need all of these functions to be localized

Reply:
I forgot to add the suggestion here but if you make global to local functions make sure to camalCase them

Reply:
Also done. Fixed as few others where the camelcase rule wasn't applied

Reply:
Is there a preference for net events to be lowercase or camelCase?

Reply:
camelCase